receiver/influxdbreceiver/receiver.go

```diff
@@ -201,6 +212,23 @@ func (r *metricsReceiver) handleWrite(w http.ResponseWriter, req *http.Request)
 		return
 	}
 
+	if len(failureReason) > 0 {
+		totalErrors := 0
+		for _, count := range failureReason {
+			totalErrors += count
+		}
+		w.WriteHeader(http.StatusBadRequest)
+		w.Header().Set("Content-Type", "application/json")
+		response := map[string]any{
+			"total_success":  line - totalErrors,
+			"total_failure":  totalErrors,
+			"failure_reason": failureReason,
```

Comment:
I'd also maybe be a nit-picking nancy and suggest this to be `failure_reasons`?
```suggestion
			"failure_reasons": failureReason,
```